test/integration/Cloning.test.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  KitchenSink,
+  PrimitiveUrl,
+  CodeableConcept,
+  PrimitiveString,
+  Coding,
+  PrimitiveCode,
+  PrimitiveTime,
+  PrimitiveBoolean,
+  IElement,
+} from "../fixture/generatedTypeScript/fhir";
+
+describe("Test cloning of the generated classes", () => {
+  let sink: KitchenSink;
+
+  beforeEach(() => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=6063602)